app/src/main/java/org/opencord/aaa/impl/AaaManager.java

```diff
@@ -345,98 +349,101 @@ protected void sendRadiusPacket(RADIUS radiusPacket, InboundPacket inPkt) {
-            default:
-                log.warn("Unknown RADIUS message received with code: {}", radiusPacket.getCode());
-                aaaStatisticsManager.getAaaStats().increaseUnknownTypeRx();
+        case RADIUS.RADIUS_CODE_ACCESS_CHALLENGE:
+            log.debug("RADIUS packet: RADIUS_CODE_ACCESS_CHALLENGE");
+            RADIUSAttribute radiusAttrState = radiusPacket.getAttribute(RADIUSAttribute.RADIUS_ATTR_STATE);
+            byte[] challengeState = null;
+            if (radiusAttrState != null) {
+                challengeState = radiusAttrState.getValue();
+            }
+            eapPayload = radiusPacket.decapsulateMessage();
+            stateMachine.setChallengeInfo(eapPayload.getIdentifier(), challengeState);
+            eth = buildEapolResponse(stateMachine.supplicantAddress(), MacAddress.valueOf(nasMacAddress),
+                    stateMachine.vlanId(), EAPOL.EAPOL_PACKET, eapPayload, stateMachine.priorityCode());
+            log.debug("Send EAP challenge response to supplicant {}", stateMachine.supplicantAddress().toString());
+            sendPacketToSupplicant(eth, stateMachine.supplicantConnectpoint());
+            aaaStatisticsManager.getAaaStats().increaseChallengeResponsesRx();
+            break;
+        case RADIUS.RADIUS_CODE_ACCESS_ACCEPT:
+            log.debug("RADIUS packet: RADIUS_CODE_ACCESS_ACCEPT");
+            // send an EAPOL - Success to the supplicant.
+            byte[] eapMessageSuccess = radiusPacket.getAttribute(RADIUSAttribute.RADIUS_ATTR_EAP_MESSAGE).getValue();
+            eapPayload = EAP.deserializer().deserialize(eapMessageSuccess, 0, eapMessageSuccess.length);
+            eth = buildEapolResponse(stateMachine.supplicantAddress(), MacAddress.valueOf(nasMacAddress),
+                    stateMachine.vlanId(), EAPOL.EAPOL_PACKET, eapPayload, stateMachine.priorityCode());
+            log.info("Send EAP success message to supplicant {}", stateMachine.supplicantAddress().toString());
+            sendPacketToSupplicant(eth, stateMachine.supplicantConnectpoint());
+
+            stateMachine.authorizeAccess();
+            aaaStatisticsManager.getAaaStats().increaseAcceptResponsesRx();
+            break;
+        case RADIUS.RADIUS_CODE_ACCESS_REJECT:
+            log.debug("RADIUS packet: RADIUS_CODE_ACCESS_REJECT");
+            // send an EAPOL - Failure to the supplicant.
+            byte[] eapMessageFailure;
+            eapPayload = new EAP();
+            RADIUSAttribute radiusAttrEap = radiusPacket.getAttribute(RADIUSAttribute.RADIUS_ATTR_EAP_MESSAGE);
+            if (radiusAttrEap == null) {
+                eapPayload.setCode(EAP.FAILURE);
+                eapPayload.setIdentifier(stateMachine.challengeIdentifier());
+                eapPayload.setLength(EAP.EAP_HDR_LEN_SUC_FAIL);
+            } else {
+                eapMessageFailure = radiusAttrEap.getValue();
+                eapPayload = EAP.deserializer().deserialize(eapMessageFailure, 0, eapMessageFailure.length);
```

Comment:
Why have you changed the formatting of all these lines??? 

Unnecessarily showing as the changes lines although there is no change